.gitignore

```diff
@@ -34,6 +34,7 @@ test_results
 /doc/ui/portfolio
 
 /vendor/cmep
+/vendor/bundle
```

Comment:
Not sure. But the 'error' (in quotes, since it is not really an error - the task exited with 0 status) was gone after I ignored the files.